### ISSUE TITLE
Menu Bar Drop-Down Menus Switch Between Dark and Light Theme

### ISSUE THREAD

**Issue by VoidTwo** [VoidTwo/Dark-Theme-Chroma-Firefox#2]
## Issue
Description: Menu Bar drop-down menus continuously switch between dark and light theme.

Applies to all Menu Bar drop-down menus (File, Edit, View, History, Bookmarks, Tools, and Help).

This is an annoying visual bug and there's nothing in the code that contributes to this.

**Comment by VoidTwo** [comment]
I have found the cause of this issue, and it is actually not random at all. After watching this closely, I noticed that the light theme appears when the chroma color is dark and switches back to dark theme (the way it should be) when the chroma color is lighter. It is very precise when it switches and I believe Firefox has a built-in measure to calculate the overall brightness of the colors. It seems that if the color is less than a sum of 355 (adding all the RGB colors) then it will switch to the light theme. To solve this I will either wait for Firefox to patch this or find a way to override this behavior.

If this bug annoys you, I have created a pastel version of this add-on and it does not have this weird bug: link [here](https://github.com/VoidTwo/Dark-Theme-Pastel-Chroma-Firefox).
